packages/plugins/geth/src/devtxs.ts

```diff
@@ -74,6 +80,10 @@ export default class DevTxs {
     if (!this.web3) {
       return;
     }
+    if (!(this.blockchainConfig.enabled && this.blockchainConfig.clientConfig && this.blockchainConfig.clientConfig.miningMode === 'dev')) {
+      return;
```

Comment:
It might be better to pull this out in `plugins/geth/src/index.js`, so that the entire devtxs functionality is not registered unless we're in --dev mode. This would prevent any `devtxs` commands from appearing in `--help` when in fact we would not able to use it, ie:
```
// plugins/geth/src/index.js, line 45

if (this.blockchainConfig.enabled && this.blockchainConfig.clientConfig && this.blockchainConfig.clientConfig.miningMode === 'dev') {
  this.setupDevTxs();
}
```